skimage/util/tests/test_dtype.py

```diff
@@ -15,62 +15,65 @@
                np.float64: (-1.0, 1.0)}
 
 
+img_funcs_and_types = {img_as_int: np.int16,
```

Comment:
Maybe change to a tuple or use `zip` instead?

Reply:
OK, I'll do it.